### PR TITLE
Rewrite/remove

### PR DESCRIPTION
**Pull request by jmwall24**
Cleaning up some topics that needed work. Collab with @multimokia.

#3803

### CODE REVIEW

**Comment by ThePotatoGuy** [comment]
@multimokia @jmwall24 can you guys list out what needs update scripts?

**Comment by jmwall24** [comment]
`monika_szs` is now `mas_story_szs` so need data transfer and unlock of the story for those who have seen it.

Derandom `monika_familygathering` for those who answered they don't have a family.

`mas_d25_monika_sleigh` is now `monika_sleigh` and needs it's data transferred.

Anything else @multimokia?

**Comment by multimokia** [comment]
Nope, that's all.

**Comment by ThePotatoGuy** [comment]
is `monika_szs` even worth keeping? its a weird one that I am willing to just drop entirely.

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -1213,26 +1194,36 @@ label monika_middleschool:
...
 
 label monika_outfit:
-    m 1lsc "You know, I'm kind of jealous that everyone else in the club had scenes outside of school..."
-    m 1lfc "That makes me the only one who hasn't gotten to dress in anything but our school uniform."
-    m 2euc "It's kind of a shame..."
-    m 2eka "I would have loved to wear some cute clothes for you."
-    m 2eua "Do you know any artists?"
-    m "I wonder if anyone would ever want to draw me wearing something else..."
-    m 2hua "That would be amazing!"
+    if len(store.mas_selspr.CLOTH_SEL_SL) == 1:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: this list stores all selectables, including locked ones. should be:
```suggestion
    if len(store.mas_selspr.filter_clothes(True)) == 1:
```

**Comment by multimokia** [comment]
Where it is now, I think it's fine, since it's no longer going to be a random topic. So I don't really have a problem with keeping it. Removal is fine too. I don't have an opinion either way.

**Comment by ThePotatoGuy** [comment]
Can you guys add #1659 in here, too? just change the button name

**Pull request merged**